src/plugins/core/range.ts

```diff
@@ -399,6 +415,30 @@ export class RangeAdapter implements CommandHandler<CoreCommand> {
     return { _sheetId: sheetId, _zone: zone };
   }
 
+  getRangeFromZone(
+    sheetId: UID,
+    zone: Zone,
+    parts: RangePart[],
```

Comment:
might as well make the arg optional, since we only ever use the value `[{ colFixed: false, rowFixed: false },{ colFixed: false, rowFixed: false },]`